test/e2e_node/init_containers_test.go

```diff
@@ -38,99 +38,131 @@ import (
 
 type containerType byte
 
-const (
-	containerTypeInvalid containerType = iota
-	containerTypeInit
-	containerTypeRegular
-)
+var defaultResourceRequirements v1.ResourceRequirements = v1.ResourceRequirements{
+	Requests: v1.ResourceList{
+		v1.ResourceMemory: resource.MustParse("15Mi"),
+	},
+	Limits: v1.ResourceList{
+		v1.ResourceMemory: resource.MustParse("15Mi"),
+	},
+}
 
-type containerTestConfig struct {
-	Name     string
-	Type     containerType
+type execCommand struct {
 	ExitCode int
 	Delay    int
 }
 
-func (c containerTestConfig) Command() []string {
+func ExecCommand(name string, c execCommand) []string {
 	var cmd bytes.Buffer
 	// all outputs are in the format of:
-	// timestamp container-name message
-	fmt.Fprintf(&cmd, "echo `date +%%s` '%s Starting' >> /shared/output; ", c.Name)
-	fmt.Fprintf(&cmd, "echo `date +%%s` '%s Delaying %d' >> /shared/output; ", c.Name, c.Delay)
+	// time-since-boot timestamp container-name message
+
+	// The busybox time command doesn't support sub-second display. uptime displays in hundredths of a second, so we
+	// include both and use time since boot for relative ordering
+	timeCmd := "`date +%s` `cat /proc/uptime | awk '{print $1}'`"
+	fmt.Fprintf(&cmd, "echo %s '%s Starting' >> /dev/termination-log; ", timeCmd, name)
+	fmt.Fprintf(&cmd, "echo %s '%s Delaying %d' >> /dev/termination-log; ", timeCmd, name, c.Delay)
 	if c.Delay != 0 {
 		fmt.Fprintf(&cmd, "sleep %d; ", c.Delay)
 	}
-	fmt.Fprintf(&cmd, "echo `date +%%s` '%s Exiting' >> /shared/output; ", c.Name)
+	fmt.Fprintf(&cmd, "echo %s '%s Exiting' >> /dev/termination-log; ", timeCmd, name)
```

Comment:
found the problem of this approach. It doesn't allow to easily check that the container was restarted. Need to check for `restartCount`

Reply:
@tzneal FYI

Reply:
for scenarios when validating that with restartPolicy Always containers will be restarted. With the host temp file it was easier

Reply:
Ah, wonder if we can:
- write to a file in ephemeral storage
- as a last step, cat that to the termination log

I think that would let you accumulate logs across restarts.

Reply:
I'll try that approach later tonight if you don't get around to it.

Reply:
yeah, interesting idea. I think this will be better than checking `restartCount`. I'll try this tonight

Reply:
I might not get to it. So you can go ahead. One problem is that PRs are super slow to merge. Hard to synchronize efforts

Reply:
Yeah, I noticed that test fix has been stuck for two hours now. I was hoping it would clear and merge overnight.

Reply:
The technique seems to work in my testing, feel free to just patch this into your branch if you want to avoid delays.  https://github.com/tzneal/kubernetes/commit/899430aa43817aa840f7e179b9e76fa3d2c1ff3a

Reply:
I am trying to get `LastTerminationStatus` but it still not working.